### ISSUE TITLE
paths with spaces

### ISSUE THREAD

**Issue by hissssst** [Parquery/pylddwrap#25]
https://github.com/thiagokokada/nix-alien/issues/31

Somehow, author refuses to open an issue

**Comment by mristin** [comment]
@hissssst thanks for posting! Could you please copy/paste the exact output so that I can add it to the regression tests?

**Comment by hissssst** [comment]
```
ldd CoQ
	linux-vdso.so.1 (0x00007ffca2b70000)
	UnityPlayer.so => /home/flynn/games/q u d/./UnityPlayer.so (0x00007f90f290f000)
	libm.so.6 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib/libm.so.6 (0x00007f90f282f000)
	libgcc_s.so.1 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib/libgcc_s.so.1 (0x00007f90f2815000)
	libpthread.so.0 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib/libpthread.so.0 (0x00007f90f2810000)
	libc.so.6 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib/libc.so.6 (0x00007f90f2606000)
	libdl.so.2 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib/libdl.so.2 (0x00007f90f25ff000)
	librt.so.1 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib/librt.so.1 (0x00007f90f25fa000)
	/lib64/ld-linux-x86-64.so.2 => /nix/store/bzd91shky9j9d43girrrj6vmqlw7x9m8-glibc-2.35-163/lib64/ld-linux-x86-64.so.2 (0x00007f90f49c3000)
```

**Comment by mristin** [comment]
Fixed by #26. A patch version is coming very soon.
